.github/workflows/CI.yml

```diff
@@ -12,15 +15,26 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          # Test on the LTS, Invenia prod release and latest release
-          - 1.0.5
-          - 1.3
           - 1
         os:
           - ubuntu-latest
+          - macOS-latest
+          - windows-latest
```

Comment:
We don't care about testing on mac or windows because the package isn't platform specific. If anything we might want to test on x86